### ISSUE TITLE
ts: theme type required errors

### ISSUE THREAD

**Issue by atanasster** [grommet/grommet#3482]
In the new theme types, none of the properties should be a required field (marginally only the brand color could be a required field).

Currently all the properties are marked as required and the theme type is unusable to create a new theme.

following generates an error
```
 <Grommet
          theme={deepMerge(theme, {
            global: {
              font: {
                family: fontString(font),
              },
            },
          })}
        >
```

https://github.com/grommet/grommet/blob/d8f7eafcccaa38b79d810d36f8f9734bf0636cd4/src/js/themes/base.d.ts#L49

**Comment by quinnturner** [comment]
I can confirm that this issue and https://github.com/grommet/grommet/issues/3481 are **breaking issues** in the new releases.

**Comment by ShimiSun** [comment]
Released hotfix on 2.7.11, thanks for reporting https://github.com/grommet/grommet/releases/tag/v2.7.11
